apps/omg_watcher/lib/block_getter/core.ex

```diff
@@ -129,29 +127,29 @@ defmodule OMG.Watcher.BlockGetter.Core do
   end
 
   def get_blocks_to_consume(
-        %__MODULE__{blocks_to_consume: blocks} = state,
+        %__MODULE__{blocks_to_consume: blocks, block_interval: interval, last_consumed_block: last_consumed_block} =
```

Comment:
This function definitely deserves a doc :)

Reply:
done